content/5_Plat/Range_Sweep.mdx

```diff
@@ -39,17 +35,18 @@ upper $x$ coordinate of the segment.
 Our answer would be the sum of all the queries.
 
 <LanguageSection>
-
 <CPPSection>
 
 ```cpp
 #include <bits/stdc++.h>
 using namespace std;
 
-int bit[2000005];
+// max coordinate
+const int X = 1e6;
+int bit[2 * X + 1];
 
 void update(int i, int x) {
-	for (; i < 2000005; i += i & (-i)) bit[i] += x;
```

Comment:
for the bit & segtree, can you use the templates [here](https://usaco.guide/gold/PURS?lang=cpp) for the sake of consistency

Reply:
i made the change

Reply:
also what is the O(what) thing

Reply:
yea i did that